build.gradle

```diff
@@ -182,26 +182,28 @@ dependencies {
     compile 'com.fasterxml.jackson.jaxrs:jackson-jaxrs-json-provider'
     compile 'com.fasterxml.jackson.module:jackson-module-jaxb-annotations'
     compile 'com.fasterxml.jackson.dataformat:jackson-dataformat-cbor'
-    compile 'com.mysema.querydsl:querydsl-jpa'
+    compile 'com.querydsl:querydsl-jpa:4.1.4'
 
     jacksonCoreLibs 'com.fasterxml.jackson.core:jackson-databind'
     jacksonCoreLibs 'com.fasterxml.jackson.core:jackson-core'
     jacksonCoreLibs 'com.fasterxml.jackson.core:jackson-annotations'
     jacksonCoreLibs 'com.fasterxml.jackson.dataformat:jackson-dataformat-cbor'
 
-    querydsl 'com.mysema.querydsl:querydsl-apt'
+    querydsl 'com.querydsl:querydsl-apt:4.1.4'
 
     /*
         Spring boot 1.3.X relies on hibernate 4 which is not compatible
         with the version that is used with Scheduling project
      */
-    compile 'org.springframework.boot:spring-boot-starter-data-jpa'
+    compile 'org.springframework.boot:spring-boot-starter-data-jpa:1.5.22.RELEASE'
 
     compile('org.hibernate:hibernate-entitymanager')
     compile('org.hibernate:hibernate-core')
+    compile('javax.validation:validation-api:1.1.0.Final')
 
-    compile 'org.springframework.boot:spring-boot-starter-log4j2'
-    compile 'org.springframework.boot:spring-boot-starter-web'
+    compile 'org.springframework.boot:spring-boot-starter-log4j2:1.5.22.RELEASE'
+    compile ('org.springframework.boot:spring-boot-starter-web:1.5.22.RELEASE')
+    compile ('org.springframework.boot:spring-boot-starter-validation:1.5.22.RELEASE')
 
```

Comment:
Is it possible to use the parent-bom for the ctalaog project?

Reply:
@nebilbenmabrouk it is using parent-bom, but when I do an upgrade of a library like this. It is not possible to use it.
The version change requires many code changes and parent bom still advertise spring boot  version 1.3.8.